python/lsst/ts/standardscripts/base_focus_sweep.py

```diff
@@ -120,6 +120,12 @@ def get_schema(cls) -> dict:
                 description: Number of steps to take inside the focus window.
                 type: number
                 minimum: 3
+              focus_steps:
+                description: Steps to take for the focus sweep.
+                type: array
+                items:
+                  type: number
+                minItems: 3
```

Comment:
I think technically minItems could be 2.

Reply:
Ok. I copied this value from ``n_steps``. Should I change ``minItems`` for ``n_steps`` too?

Reply:
Sure, I think that makes sense.